test/ng/compileSpec.js

```diff
@@ -12962,6 +12962,27 @@ describe('$compile', function() {
       });
 
     });
+
+    describe('*[style]', function() {
+      it('should NOT set style for untrusted values', inject(function($rootScope, $compile) {
+        element = $compile('<div ng-prop-style="style"></div>')($rootScope);
+        $rootScope.style = 'background: red';
+        expect(function() { $rootScope.$digest(); }).toThrow();
```

Comment:
Should we be more strict on what is thrown here? E.g. `.toThrowMinErr('$sce', 'insecurl', ...)`?

Reply:
Done (b3563c58db80721795542ec8d9e27ec0936e1d42), but note it is a `unsafe` error in these cases.